.devcontainer/Dockerfile

```diff
@@ -25,3 +25,4 @@ ENV PATH="~/.poetry/bin:$PATH"
 ENV PATH="~/.local/bin:$PATH"
 
 RUN /usr/local/bin/python3 -m pip install -U flake8
+RUN /usr/local/bin/python3 -m pip install -U black
```

Comment:
Taking a look at the qaqc dockerfile, we install bandit and not flake8. I think it makes sense to stay as consistent as possible

Reply:
Rebuilding the dev container and it's taking ages, is this true for you as well @SashaWeinstein

Reply:
started rebuilding now

Reply:
got it finally

Reply:
Mine rebuilt pretty quick, meant to look at clock but within 5 minutes I think. Max when was the last time you turned off your computer? 
Also made the requested change from `flake8` to `bandit`

Reply:
yesterday. great will rebuild.